Update pytorch-train-model.md

nonmanifold opened the pull request:
fix testAccuracy() code to be able to run on cuda and correct reference to in entry point
fixes #37

opbld33 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/01samiksha/windows-ai-docs/blob/patch-1/docs/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/index?branch=pr-en-us-42) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C11%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201810111826215522-42%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=MAuFJU9b7OgBfC9Zc2SJbjxNGvrrEBRPxh7xtmUcPPc%3D&st=2018-10-11T18%3A22%3A34Z&se=2018-11-11T18%3A27%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

nonmanifold committed:
Update pytorch-train-model.md

fix testAccuracy() code to be able to run on cuda and correct reference to in entry point
